packages/integration-sdk-core/src/types/config.ts

```diff
@@ -86,6 +90,14 @@ export interface InvocationConfig<
    *
    */
   ingestionConfig?: IntegrationIngestionConfigFieldMap;
+  /**
+   * Wraps the executionHandler for each step in an operation to allow for adding
+   * context before and after the executionHandler completes. Can be used for adding
+   * logic like tracing or logging.
+   *
+   * If not provided, this defaults to a noop wrapper.
```

Comment:
Biggest nitpick of the century: If I understand correctly, looking at line 358 in dependancyGraph.ts, if no wrapper is provided we just run the handler as is; not run a noop wrapper. So these docs _could_ be updated but it's really not a big deal.

Reply:
It's worth fixing so it doesn't cause confusion later, will update accordingly